client/tests/e2e/specs/test_greet_user_modal.ts

```diff
@@ -15,7 +15,7 @@ describe('Greet user into an organization', () => {
     cy.dropTestbed();
   });
 
-  it('Open greet user modal', () => {
+  it.skip('Open greet user modal', () => {
```

Comment:
why skip it?

Reply:
So the tests would run a little faster while debugging, forgot to remove it. Good catch!